build.gradle.kts

```diff
@@ -44,3 +45,17 @@ dependencies {
 	testImplementation("org.springframework:spring-webmvc:2.5.2")
 	testImplementation("org.springframework:spring-mock:2.0.7")
 }
+
+publishing {
+	repositories {
+		maven {
+			name = "GitHubPackages"
+			url = uri("https://maven.pkg.github.com/infonl/webdav-servlet")
+			version = "2.1-dev"
```

Comment:
Overschrijft ie de bestaande versie? We gaan het wel zien I guess.. ;-)

Reply:
In mijn ervaring met GH packages wel ja, daarom werkt `latest` ook